styles/electrified.mapcss

```diff
@@ -341,12 +343,14 @@ node|z17-[railway=signal]["railway:signal:electricity"="DE-AVG:st7"]["railway:si
 	allow-overlap: true;
 }
 
-/**************************/
+/*************************/
 /* DE power on sign El 2 */
-/* AT Hauptschalter ein   */
-/**************************/
+/* AT Hauptschalter ein  */
+/* NL inschakelbord 307a */
+/*************************/
 node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:electricity"="DE-ESO:el2"]["railway:signal:electricity:type"="power_on"]["railway:signal:electricity:form"="sign"],
 node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:electricity"="AT-V2:hauptschalter_ein"]["railway:signal:electricity:type"="power_on"]["railway:signal:electricity:form"=sign]
```

Comment:
a comma is missing at the end of this line

Reply:
resolved